lib/cc/analyzer/issue.rb

```diff
@@ -24,7 +24,7 @@ def as_json(*)
       end
 
       def fingerprint
-        parsed_output.fetch("fingerprint", default_fingerprint)
+        parsed_output["fingerprint"] || default_fingerprint
```

Comment:
Ah sorry. How about `parsed_output.fetch("fingerprint") { default_fingerprint }` (the block is only evaluated on missing keys)?